packages/turf-point-grid/index.d.ts

```diff
@@ -3,6 +3,6 @@ import {BBox, Points, Units, Feature, Features} from '@turf/helpers';
 /**
  * http://turfjs.org/docs/#pointgrid
  */
-declare function pointGrid(bbox: BBox | Feature<any> | Features<any>, cellSide: number, units?: Units, centered?: boolean, bboxIsMask?: boolean): Points;
+declare function pointGrid(bbox: BBox | Feature<any> | Array<number>, cellSide: number, units?: Units, centered?: boolean, bboxIsMask?: boolean): Points;
```

Comment:
@stebogit 1. Why the removal of `Features<any>` this is to support FeatureCollection, I guess we should use the full name of the Type (`FeatureCollection<any>`) instead of making the types plural.

2. BBox is already defined - `Array<number>` is not needed

`BBox` => `[number, number, number, number]`